error: ‘nullptr’ was not declared in this scope

Hi, when I build OpenCV with the module contrib I got the following mistake : error: ‘nullptr’ was not declared in this scope (see below for details).

If someone has and idea that would be very helpful.
Thanks

```
[ 79%] Generating test_precomp.hpp.gch/opencv_test_cvv_RELEASE.gch
In file included from /net/rmc-lx0075/home_local/OpenCV_Contrib/opencv_contrib-master/modules/cvv/include/opencv2/cvv.hpp:19:0,
                 from /home_local/OpenCV_Contrib/opencv-3.1.0/build/modules/cvv/test_precomp.hpp:15:
/net/rmc-lx0075/home_local/OpenCV_Contrib/opencv_contrib-master/modules/cvv/include/opencv2/cvv/call_meta_data.hpp: In constructor ‘cvv::impl::CallMetaData::CallMetaData()’:
/net/rmc-lx0075/home_local/OpenCV_Contrib/opencv_contrib-master/modules/cvv/include/opencv2/cvv/call_meta_data.hpp:27:13: error: ‘nullptr’ was not declared in this scope
      : file(nullptr), line(0), function(nullptr), isKnown(false)
             ^
In file included from /net/rmc-lx0075/home_local/OpenCV_Contrib/opencv_contrib-master/modules/cvv/include/opencv2/cvv.hpp:21:0,
                 from /home_local/OpenCV_Contrib/opencv-3.1.0/build/modules/cvv/test_precomp.hpp:15:
/net/rmc-lx0075/home_local/OpenCV_Contrib/opencv_contrib-master/modules/cvv/include/opencv2/cvv/dmatch.hpp: At global scope:
/net/rmc-lx0075/home_local/OpenCV_Contrib/opencv_contrib-master/modules/cvv/include/opencv2/cvv/dmatch.hpp:83:47: error: ‘nullptr’ was not declared in this scope
                                const char * = nullptr, const char * = nullptr,
                                               ^
/net/rmc-lx0075/home_local/OpenCV_Contrib/opencv_contrib-master/modules/cvv/include/opencv2/cvv/dmatch.hpp:83:71: error: ‘nullptr’ was not declared in this scope
                                const char * = nullptr, const char * = nullptr,
                                                                       ^
In file included from /net/rmc-lx0075/home_local/OpenCV_Contrib/opencv_contrib-master/modules/cvv/include/opencv2/cvv.hpp:22:0,
                 from /home_local/OpenCV_Contrib/opencv-3.1.0/build/modules/cvv/test_precomp.hpp:15:
/net/rmc-lx0075/home_local/OpenCV_Contrib/opencv_contrib-master/modules/cvv/include/opencv2/cvv/filter.hpp:61:47: error: ‘nullptr’ was not declared in this scope
                                const char * = nullptr, const char * = nullptr)
                                               ^
/net/rmc-lx0075/home_local/OpenCV_Contrib/opencv_contrib-master/modules/cvv/include/opencv2/cvv/filter.hpp:61:71: error: ‘nullptr’ was not declared in this scope
                                const char * = nullptr, const char * = nullptr)
                                                                       ^
In file included from /net/rmc-lx0075/home_local/OpenCV_Contrib/opencv_contrib-master/modules/cvv/include/opencv2/cvv.hpp:24:0,
                 from /home_local/OpenCV_Contrib/opencv-3.1.0/build/modules/cvv/test_precomp.hpp:15:
/net/rmc-lx0075/home_local/OpenCV_Contrib/opencv_contrib-master/modules/cvv/include/opencv2/cvv/show_image.hpp:62:45: error: ‘nullptr’ was not declared in this scope
                              const char * = nullptr, const char * = nullptr)
                                             ^
/net/rmc-lx0075/home_local/OpenCV_Contrib/opencv_contrib-master/modules/cvv/include/opencv2/cvv/show_image.hpp:62:69: error: ‘nullptr’ was not declared in this scope
                              const char * = nullptr, const char * = nullptr)
                                                                     ^
modules/cvv/CMakeFiles/pch_Generate_opencv_test_cvv.dir/build.make:62: recipe for target 'modules/cvv/test_precomp.hpp.gch/opencv_test_cvv_RELEASE.gch' failed
make[2]: *** [modules/cvv/test_precomp.hpp.gch/opencv_test_cvv_RELEASE.gch] Error 1
CMakeFiles/Makefile2:11757: recipe for target 'modules/cvv/CMakeFiles/pch_Generate_opencv_test_cvv.dir/all' failed
make[1]: *** [modules/cvv/CMakeFiles/pch_Generate_opencv_test_cvv.dir/all] Error 2
Makefile:149: recipe for target 'all' failed
make: *** [all] Error 2
```

--- a/modules/cvv/CMakeLists.txt
+++ b/modules/cvv/CMakeLists.txt
@@ -4,7 +4,7 @@ if(NOT HAVE_QT5)
 endif()
 
 set(the_description "Debug visualization framework")
-ocv_define_module(cvv opencv_core opencv_imgproc opencv_features2d ${CVV_LIBRARIES} WRAP python)
+ocv_add_module(cvv opencv_core opencv_imgproc opencv_features2d WRAP python)
 
 # we need C++11 and want warnings:
 if(MSVC)
@@ -23,3 +23,10 @@ foreach(dt5_dep Core Gui Widgets)
   include_directories(${Qt5${dt5_dep}_INCLUDE_DIRS})
   list(APPEND CVV_LIBRARIES ${Qt5${dt5_dep}_LIBRARIES})
 endforeach()
+
+ocv_glob_module_sources()
+ocv_module_include_directories()
+ocv_create_module(${CVV_LIBRARIES})
+ocv_add_accuracy_tests()
+ocv_add_perf_tests()
+ocv_add_samples()

